federation/manifests/federation-apiserver-deployment.yaml

```diff
@@ -22,7 +22,11 @@ spec:
         - --etcd-servers=http://localhost:2379
         - --service-cluster-ip-range={{.FEDERATION_SERVICE_CIDR}}
         - --secure-port=443
+        {{if eq .KUBERNETES_PROVIDER "aws"}}
+        - --external-hostname={{.FEDERATION_API_HOST}}
```

Comment:
What does not work if we do not set this?
Reading the documentation, I get the impression that if its a public IP, we can set it to external-hostname.
And if its a cluster IP, we can set it to advertise-address.